modules/basic_auth/handler/basic_auth_logoff_handler.e

```diff
@@ -55,21 +55,62 @@ feature -- HTTP Methods
 			else
 				create {GENERIC_VIEW_CMS_RESPONSE} l_page.make (req, res, api)
 				unset_current_user (req)
-				l_page.set_status_code ({HTTP_STATUS_CODE}.found) -- Note: can not use {HTTP_STATUS_CODE}.unauthorized for redirection
-				if attached {WSF_STRING} req.query_parameter ("destination") as l_uri then
```

Comment:
why did you removed the redirection here?
this is quite convenient when you logoff to get redirected to previous visited page.